spec/lib/audit/moab_to_catalog_spec.rb

```diff
@@ -23,94 +33,60 @@
   end
 
   describe ".check_existence_for_all_storage_roots" do
-    let(:subject) { described_class.check_existence_for_all_storage_roots }
-
     it 'calls check_existence_for_dir once per storage root' do
       expect(described_class).to receive(:check_existence_for_dir).exactly(HostSettings.storage_roots.entries.count).times
-      subject
+      described_class.check_existence_for_all_storage_roots
```

Comment:
No objection, just curious about the motivation for moving away from using `subject` here and below?

Reply:
NM, I see your comment about it's being an antipattern, and finding discussion out there about that, for example thoughtbot advises against it.

Reply:
Right, the main thing is that this is really a timing-dependent invocation and not an object or value that is setup on demand like a normal `let` statement.  

If one wants `let(:subject) {...}`, they probably should `subject {...}`, but that would make it clear in these cases (to rubocop-rspec) that some non-kosher things are being done.